AIPscan/Reporter/templates/reports.html

```diff
@@ -65,48 +65,68 @@
+        </td>
+          <td><span class="text-muted">Filter by date range</span></td>
+        <td><a href="#"><button type="button" id="report2a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
       </tr>
 
       <tr>
-          <td>Format version count</td>
-          <td>Uses date fields</td>
-          <td><a href="#"><button type="button" id="report3a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+          <td><strong>Largest files</strong>
+            <br>
+            <span class="text-muted">Top twenty largest files in the Storage Service or Storage Location</td>
+          <td><span class="text-muted">Filter by storage location</span></td>
+          <td><a href="#"><button type="button" id="report9a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
       </tr>
 
       <tr>
-          <td>Preservation derivatives</td>
-          <td></td>
-          <td><a href="#"><button type="button" id="preservationDerivatives" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+          <td><strong>File format count</strong>
+            <br>
+            <span class="text-muted">Number of files per format and the volume of storage each uses
+            </td>
+          <td><span class="text-muted">Filter by date range
+            <br>
+            Filter by storage location</span></td>
+          <td><a href=""><button type="button" id="report1a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a><a href=""><button type="button" id="report1b" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Pie Chart</button></a><a href=""><button type="button" id="report1c" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Scatter Plot</button></a></td>
       </tr>
 
       <tr>
-          <td>Largest files</td>
-          <td></td>
-          <td><a href="#"><button type="button" id="report9a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+          <td><strong>Format version count</strong>
+            <br>
+            <span class="text-muted">Number of files per format version (PUID) and the volume of storage each uses</td>
+          <td><span class="text-muted">Filter by date range
+            <br>
+            Filter by storage location</span></td>
+          <td><a href="#"><button type="button" id="report3a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
       </tr>
 
       <tr>
           <td>
-              File format
+              <strong>AIPs by file format</strong>
               <br>
-              <span class="text-muted">AIPs containing original files in given file format.</span>
+              <span class="text-muted">AIPs containing original files in the selected format, filter by Storage Location</span>
```

Comment:
I find the difference between the location of "Filter by Storage Location" in these AIPs by x reports vs. their inclusion in the "Parameters" column for other reports a bit jarring. Could we add the note above or below the dropdown maybe? Or just elide it, per my main comment?